lib/event-highlighting/event-map-helper.js

```diff
@@ -0,0 +1,59 @@
+'use babel'
+
+class EventMapHelper {
```

Comment:
I admit, I'm not the biggest fan of "helpers" and "utils" classes, so I'd suggest to avoid them as much as possible.

In this case, both the functions are only used by the worker, so could them be moved there?